manifests/y/Yemiao/Xtranslator/2.5.6/Yemiao.Xtranslator.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=QUSU.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Yemiao.Xtranslator
+PackageVersion: 2.5.6
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+ReleaseDate: 2022-12-07
+Installers:
+- Architecture: x86
+  InstallerUrl: https://note.youdao.com/yws/api/personal/file/WEB583211904592ae7352f9bcb04c33da74?method=download&shareKey=d9cd7960c56edbae0054b31eba4715d7
```

Comment:
I wasn't able to find this InstallerUrl from the PackageUrl - it doesn't match any of the publisher's approved mirrors found on http://www.zhiyunwenxian.cn/winx.html.

Reply:
cc @SpectorShell